src/main/java/co/rsk/federate/config/PowpegNodeSystemProperties.java

```diff
@@ -0,0 +1,137 @@
+package co.rsk.federate.config;
+
+import static co.rsk.federate.config.PowpegNodeConfigParameter.*;
+
+import co.rsk.config.ConfigLoader;
+import co.rsk.config.RskSystemProperties;
+import co.rsk.federate.signing.config.SignerConfig;
+import com.typesafe.config.Config;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+public class PowpegNodeSystemProperties extends RskSystemProperties {
+
+  public static final String REGTEST = "regtest";
+
+  public PowpegNodeSystemProperties(ConfigLoader loader) {
+    super(loader);
+  }
+
+  public boolean isFederatorEnabled() {
+    return getBoolean(
+        FEDERATOR_ENABLED.getPath(),
+        FEDERATOR_ENABLED.getDefaultValue(Boolean::parseBoolean));
+  }
+
+  public boolean isPegoutEnabled() {
+    return getBoolean(
+        PEGOUT_ENABLED.getPath(),
+        PEGOUT_ENABLED.getDefaultValue(Boolean::parseBoolean));
+  }
+
+  public boolean isUpdateBridgeTimerEnabled() {
+    return !netName().equals(REGTEST) ||
+        getBoolean(
+            UPDATE_BRIDGE_TIMER_ENABLED.getPath(),
+            UPDATE_BRIDGE_TIMER_ENABLED.getDefaultValue(Boolean::parseBoolean));
+  }
+
+  public boolean shouldUpdateBridgeBtcBlockchain() {
+    return getBoolean(
+        UPDATE_BRIDGE_BTC_BLOCKCHAIN.getPath(),
+        UPDATE_BRIDGE_BTC_BLOCKCHAIN.getDefaultValue(Boolean::parseBoolean));
+  }
+
+  public boolean shouldUpdateBridgeBtcCoinbaseTransactions() {
+    return getBoolean(
+        UPDATE_BRIDGE_BTC_COINBASE_TRANSACTIONS.getPath(),
+        UPDATE_BRIDGE_BTC_COINBASE_TRANSACTIONS.getDefaultValue(Boolean::parseBoolean));
+  }
+
+  public boolean shouldUpdateBridgeBtcTransactions() {
+    return getBoolean(
+        UPDATE_BRIDGE_BTC_TRANSACTIONS.getPath(),
+        UPDATE_BRIDGE_BTC_TRANSACTIONS.getDefaultValue(Boolean::parseBoolean));
+  }
+
+  public boolean shouldUpdateCollections() {
+    return getBoolean(
+        UPDATE_COLLECTIONS.getPath(),
+        UPDATE_COLLECTIONS.getDefaultValue(Boolean::parseBoolean));
+  }
+
+  public int getAmountOfHeadersToSend() {
+    return getInt(
+        AMOUNT_HEADERS.getPath(),
+        AMOUNT_HEADERS.getDefaultValue(Integer::parseInt));
+  }
+
+  /**
+   * 6000 blocks is 150% the amount of blocks the Bridge waits before confirming a
+   * peg-out.
+   * If this powpeg-node was shutdown for 48hs this depth will be enough to resync
+   * all the information.
+   * If this powpeg-node was shutdown for longer periods, most likely the
+   * transaction was signed by other functionaries.
+   */
```

Comment:
Not sure this is the right place for this comment. It should actually go where the default value is defined

Reply:
Updated